server/container/util/src/test/resources/docker/spamassassin/Dockerfile

```diff
@@ -9,12 +9,24 @@ RUN apt-get update && \
         libmail-dkim-perl \
         libnet-ident-perl \
         libsocket-getaddrinfo-perl \
+        pyzor \
+        razor \
+        libdbi-perl \
+        libdbd-pg-perl \
         spamassassin=${SPAMASSASSIN_VERSION}* && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-RUN mkdir /etc/mail/spamassassin/bayes_db && \
-    chmod -R 777 /etc/mail/spamassassin/bayes_db
+RUN mkdir -p /etc/spamassassin/sa-update-keys && \
+    chmod 700 /etc/spamassassin/sa-update-keys && \
+    chown debian-spamd:debian-spamd /etc/spamassassin/sa-update-keys && \
+    mkdir -p /var/lib/spamassassin/.pyzor && \
+    chmod 700 /var/lib/spamassassin/.pyzor && \
+    echo "public.pyzor.org:24441" > /var/lib/spamassassin/.pyzor/servers && \
```

Comment:
external access needed is back?

Reply:
The results are really unpredictable without those dependencies.